inst/templates/ad_adcm.R

```diff
@@ -16,16 +16,20 @@ library(lubridate)
 data("cm")
 data("adsl")
 data("ex")
-data("adsl")
 
 # ---- Derivations ----
+
+# Derive flags
 adcm <- cm %>%
 
   # Join supplementary qualifier variables
   # derive_vars_suppqual(suppcm)
 
-  # Join adsl to cm
-  left_join(adsl, by = c("STUDYID", "USUBJID")) %>%
+  # Join ADSL with CM (only ADSL vars required for derivations)
+  left_join(
+    adsl %>% select(STUDYID, USUBJID, TRTSDT, TRTEDT, DTHDT, EOSDT),
+    by = c("STUDYID", "USUBJID")
```

Comment:
Please don't use nested `%>%` but rather `select(adsl, STUDYID, ...)`.

Reply:
Done throughout the scripts.